src/App.jsx

```diff
@@ -14,45 +16,95 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+export const App = () => {
+  const [goods, setGoods] = useState(goodsFromServer);
+  const [isAlphabeticaly, setIsAlphabeticaly] = useState(true);
+  const [isLength, setIsLength] = useState(true);
+  const [isReverse, setReverse] = useState(true);
+
+  const sortByAlphabetically = () => {
+    const newGoods = [...goods]
+      .sort((good1, good2) => good1.localeCompare(good2));
+
+    setGoods(newGoods);
+    setIsAlphabeticaly(false);
+    setIsLength(true);
+    setReverse(true);
+  };
+
+  const sortByLength = () => {
+    const newGoods = [...goods]
+      .sort((good1, good2) => good1.length - good2.length);
+
+    setGoods(newGoods);
+    setIsLength(false);
+    setIsAlphabeticaly(true);
+    setReverse(true);
+  };
+
+  const reverseGoods = () => {
+    const newGoods = [...goods].reverse();
+
+    setGoods(newGoods);
+    setReverse(!isReverse);
+  };
 
-    <ul>
-      <li data-cy="Good">Dumplings</li>
-      <li data-cy="Good">Carrot</li>
-      <li data-cy="Good">Eggs</li>
-      <li data-cy="Good">Ice cream</li>
-      <li data-cy="Good">Apple</li>
-      <li data-cy="Good">...</li>
-    </ul>
-  </div>
-);
+  const reset = () => {
+    setGoods(goodsFromServer);
+    setIsAlphabeticaly(true);
+    setIsLength(true);
+    setReverse(true);
+  };
+
+  return (
+    <div className="section content">
+      <div className="buttons">
+        <button
+          type="button"
+          className={cn('button', 'is-info', {
+            'is-light': isAlphabeticaly,
+          })}
+          onClick={() => sortByAlphabetically()}
```

Comment:
```suggestion
          onClick={sortByAlphabetically}
```
you can just pass the name of function if it does not receive any params

Reply:
agree with [comment](https://github.com/mate-academy/react_list-of-goods-js/pull/31/files#r1247795821)
fix all similar places